examples/pytorch/AxHyperOptimizationPTL/README.md

```diff
@@ -0,0 +1,64 @@
+## Ax Hyperparameter Optimization Example 
+In this example, we train a Pytorch Lightning model to classify [CIFAR-10 dataset](https://github.com/pytorch/vision/blob/master/torchvision/datasets/cifar.py). The code, adapted from this [repository](https://pytorch.org/tutorials/beginner/blitz/cifar10_tutorial.html), is almost entirely dedicated to model training and hyperparameter optimization.
+A parent run will be created during the training process,which would dump the baseline model and relevant parameters,metrics and model along with its summary,subsequently followed by a set of nested child runs, which will dump the trial results.
+The best parameters would be dumped into the parent run once the experiments are completed.
+
+### Running the code
+To run the example via MLflow, navigate to the `mlflow/examples/pytorch/axbotorch` directory and run the command
+
+```
+mlflow run .
+```
+
+This will run `AxHyperOptimizationPTL.py` with the default set of parameters such as  `--max_epochs=3` and `total_trials=3`. You can see the default value in the `MLproject` file.
```

Comment:
```suggestion
This will run `AxHyperOptimizationPTL.py` with the default set of parameters such as  `max_epochs=3` and `total_trials=3`. You can see the default value in the `MLproject` file.
```

Reply:
Updated the line as mentioned